app/templates/patch.hbs

```diff
@@ -2,6 +2,10 @@
   <div class="file">
     <div class="meta">
       <span>{{diff.filePath}}</span>
+      <span class="links">
+        <a class="source" href="{{diff.sourceFileUrl}}" title="View generated file at source version">{{diff.sourceVersion}}</a> ...
```

Comment:
I know, and there's also a Unicode code point but, it's also a common source of debate. Checked that GitHub currently uses dots in its "version compare" feature:

```html
<span class="dots">...</span>
```

I'll get it replaced anyway.